x-pack/solutions/observability/plugins/observability/public/pages/alert_details/hooks/use_discover_url/get_rule_data/synthetics_tls_rule.ts

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
This file was already reviewed, I've only moved it to another file.
No need to review it.